src/app/components/footer/footer.html

```diff
@@ -1,38 +1,41 @@
-<footer class="p-60" style="padding-bottom: 10px">
+<footer class="p-60" style="background-color:#262626; padding-bottom: 10px">
 
-  <div layout="row" layout-xs="column" layout-sm="column" layout-wrap layout-align="space-between center" layout-align-xs="space-between stretch " style="color:white; margin:0 auto; max-width:1200px">
+    <div layout="row" layout-xs="column" layout-sm="column" layout-wrap layout-align="space-between center" layout-align-xs="space-between stretch " style="color:white; margin:0 auto; max-width:1200px">
```

Comment:
unless you are writing CSS for critical render (minimal css so that first-paint on browser is very fast), it is best not to use inline-styling.

Reply:
agree, was already there, but could be an opportunity to remove it. @pral2a what do you think?